homeassistant/components/freebox/router.py

```diff
@@ -89,7 +92,24 @@ async def update_all(self, now: datetime | None = None) -> None:
     async def update_device_trackers(self) -> None:
         """Update Freebox devices."""
         new_device = False
-        fbx_devices: list[dict[str, Any]] = await self._api.lan.get_hosts_list()
+
+        # Access to host list not available in bridge mode, API return error_code 'nodev'
+        try:
+            fbx_devices: list[dict[str, Any]] = await self._api.lan.get_hosts_list()
```

Comment:
To fix `mypy`, this var has to be initialized once.

Initialize it with an empty arr, so no need to initialize on the specific error code.
Just need to log the error.